images/core.proto

```diff
@@ -84,7 +86,7 @@ message thread_core_entry {
 	optional sint32			sched_nice	= 3;
 	optional uint32			sched_policy	= 4;
 	optional uint32			sched_prio	= 5;
-	optional uint64			blk_sigset	= 6;
+	repeated uint64			blk_sigset	= 6;
```

Comment:
Not sure if this has been discussed before. But does this break our protobuf files if we redefine an object from optional to repeated?

Reply:
I think, changing optional to repeated instead of adding a new filed - that what I've asked in the previous review. Quoting from language spec:

> `optional` is compatible with `repeated`. Given serialized data of a repeated field as input, clients that expect this field to be optional will take the last input value if it's a primitive type field or merge all input elements if it's a message type field. 

https://developers.google.com/protocol-buffers/docs/proto

Reply:
Thanks. Sounds good.